docs/Data-Integrity-Proof-Support.md

```diff
@@ -21,6 +21,60 @@ Supported signature algorithms for these crypto suites include:
 
 > **Note:** The list of supported algorithms may be extended in future releases. Always refer to the latest [API documentation](https://mosip.stoplight.io/docs/inji-certify) for updates.
 
+## Sequence Diagram for credential issuance with Data Integrity Proof
+
+```mermaid
+sequenceDiagram
+    participant Client as 🌐 Client
+    box Inji Certify #E6F3FF
+    participant CredentialAPI as 🔗 Credential API
+    participant CredentialConfiguration as ⚙️ Credential Configuration
+    participant DataProviderPlugin as 🔌 Data Provider Plugin
+    participant VelocityTemplatingEngine as ⚙️ Velocity Templating Engine
+    participant W3CJsonLdCredential as 🔐 W3CJsonLdCredential
+    participant CertifyKeyChooser as 🔑 CertifyKeyChooser
+    participant DanubetechDataIntegrity as 🛡️ Danubetech Lib
+    participant CertifyProofGenerators as 📝 Proof Generators
+    participant KeyManager as ✍️ KeyManager
+    end
+
+    Client->>CredentialAPI: Request VC Issuance (format: ldp_vc)
+
+    CredentialAPI->>CredentialConfiguration: Validate request & get config
+    CredentialConfiguration-->>CredentialAPI: Return success & config (with signatureCryptosuite)
+
+    CredentialAPI->>DataProviderPlugin: Request data
+    DataProviderPlugin-->>CredentialAPI: Return raw data
+
+    CredentialAPI->>VelocityTemplatingEngine: Format raw data with template
+    VelocityTemplatingEngine-->>CredentialAPI: Return unsigned credential data
+
+    CredentialAPI->>W3CJsonLdCredential: Instantiate with unsigned data
+
+    CredentialAPI->>W3CJsonLdCredential: addProof()
+
+    W3CJsonLdCredential->>CertifyKeyChooser: Get proof object
```

Comment:
are we called get proof object to CertifyKeyChooser ?
If no, lets remove the CertifyKeyChooser vertical line, since we might remove this in the future.